pages/dkp/kommander/2.1/security/howto-dex-saml-connector/index.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: layout.pug
+navigationTitle: SAML
+title: SAML connector
+menuWeight: 33
+excerpt: Connect your Kommander cluster to an Idp using SAML
+beta: false
+enterprise: false
+---
+
+## How to connect Kommander to an IdP using SAML
+
+This guide shows how to configure your Kommander cluster to use SAML to connect to an identity provider (IdP).
+
+Step 1: [Install Kommander](../../install).
+
+Step 2: Configure the IdP
```

Comment:
```suggestion
1.   Configure the IdP
```

Reply:
Just to double check, did you mean `2.` here instead of `1.`?

Reply:
Actually, I've just noticed that all the steps have been changed to `1.`. Can I ask why that is?

Reply:
In markdown you can number all steps with `1.`'s which automatically renders as an ordered list, which is nice so if you ever had to add a step in the middle you don't need to redo the entire numbered list https://www.markdownguide.org/basic-syntax/#ordered-lists

Reply:
Good to know! However, locally, it's not working as expected. The first two steps are numbered correctly, but the 3rd step is resetting back to 1. I'm guessing because of all the copy in between the steps. Not sure if there's a way around this or if we need to go with a different approach.

Reply:
Probably something to do with the indents and spacing to get it working properly. I can take a look at your branch to try to get it working

Reply:
That's alright, I'm sure you've got loads of your own work! You've already been super helpful. I can play around with it on Monday and see if I sort it out. Thanks for the offer though!

Reply:
@seialkali, could you please let me know if you find a solution? I am currently working on a document where I am seeing the same behavior.